src/codeu/chat/server/Server.java

```diff
@@ -53,10 +54,17 @@
   private final Timeline timeline = new Timeline();
 
   private final Map<Integer, Command> commands = new HashMap<>();
-
   private final Uuid id;
   private final Secret secret;
+  private ServerInfo info = null;
 
+  {
+    try {
+      info = new ServerInfo(Uuid.parse("1.0.0"));
```

Comment:
I moved the initialization to the constructor and created a String variable for the version, which the user can changed based on what they want the server version to be. I also added comments explaining these two and how they can be customized.